test/e2e/resources/trail.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: cloudtrail.services.k8s.aws/v1alpha1
+kind: Trail
+metadata:
+  name: $TRAIL_NAME
+spec:
+  name: $TRAIL_NAME
+  s3BucketName: $BUCKET_NAME
```

Comment:
can we test the tag-on-create by setting tags on the resource YAML from the beginning?

Reply:
Done